examples/benchmarks/query/dub.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "query",
+    "targetType": "executable",
+    "targetPath": "../../.out/bin",
+    "sourcePaths": ["source"],
+    "importPaths": ["source"],
+    "dependencies": {
+        "valhala_ecs": {"path": "../../../"}
+    },
+    "dflags": ["-release", "-O", "-inline", "-boundscheck=off"]
```

Comment:
You could make multiple `configurations` for custom flags, although, I know the purpose is only to test with those ones.

Reply:
Yeah, I just made a simple benchmark example. Maybe in the future I can do more with multiple configs.